src/ansys_sphinx_theme/__init__.py

```diff
@@ -377,8 +380,18 @@ def configure_theme_logo(app: Sphinx):
 
     if logo_option == "ansys":
         theme_options["logo"] = ansys_logo
+        theme_options["logo_link"] = (
+            ANSYS_LOGO_LINK
+            if not theme_options.get("logo_link")
+            else theme_options.get("logo_link")
+        )
```

Comment:
```suggestion
        theme_options["logo_link"] = theme_options.get("logo_link", ANSYS_LOGO_LINK)
```

Reply:
After some though, did you do the current implementation to handle the case where `theme_options["logo_link"]=""` ?